src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -129,7 +129,7 @@ class RsErrorAnnotatorTest : RsAnnotationTestBase() {
             fn variadic_2(p1: u32, p2: u32, ...);
         }
 
-        fn main() {
+        unsafe fn main() {
```

Comment:
It seems you want to avoid unnecessary error annotation that you added in this PR.
But `main` can't be unsafe, so let's rename it somehow